pom.xml

```diff
@@ -76,6 +76,24 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <version>1.17.1</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>junit</groupId>
+                    <artifactId>junit</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>mysql</artifactId>
+            <version>1.17.1</version>
```

Comment:
should that be a "test" dependency?

Reply:
This was not made a test dependency explicitly so that someone could decide to configure their local instance to run with TC as well out of the box with a configuration change, without needing to separately install MySQL.